docs/src/contracts/env-variable.md

```diff
@@ -4,47 +4,108 @@ cw-orch leverages some environment variables to interact with contracts on actua
+```
+
+## Transaction options
+
+### CW_ORCH_GAS_BUFFER
+
+Optional, accepted values: float
+
+This allows changing the gas buffer applied after tx simulation. Use this in case a transaction is blocked for insufficient gas reasons.
+
+### CW_ORCH_MIN_GAS
+
+Optional, accepted values: integer
+
+Minimum gas amount for every transaction. Useful when transaction still won't pass even when setting a high gas_buffer or for mixed transaction scripts.
+
+### CW_ORCH_MAX_TX_QUERY_RETRIES
+
+Optional, accepted values: integer
+Defaults to `50`.
+
+Changes the number of tx queries (~1 query per block) before it fails if it doesn't find any result. Useful if the chain is slow or if the transaction has low gas price.
+
+### CW_ORCH_MIN_BLOCK_SPEED
+
+Optional, accepted value: integer in seconds
+Defaults to `1`.
+
+Minimum block speed in seconds. This is used internally by `cw-orch` when broadcasting transactions. Useful when the block speeds are varying a lot
+
+### CW_ORCH_DISABLE_WALLET_BALANCE_ASSERTION
+
+Optional, accepted values: `true`, `false`
+Defaults to `false`
+
+By default, `cw-orch` verifies that the account has sufficient balance to pay for gas fees. If it detects that the balance is too low, it propmts the user to fill up their wallet with gas tokens and allows for retrying at the press of a button.
+
+If set to `true`, it won't check the user has enough balance before broadcasting transactions.
+
+### CW_ORCH_DISABLE_MANUAL_INTERACTION
+
+Optional, accepted values: `true`, `false`
+Defaults to `false`
+
+Some actions require user-intervention. If set to true, this disables those interventions. Useful when working in scripts that can't handle user-intervention.
+
+Supported actions :
+
+- Balance checks. When set to `true`, if the gas token balance is too low to submit a transaction, it will error. See [Disable balance assertion](#cw_orch_disable_wallet_balance_assertion).
+- Deployment checks. When set to `true`, if no deployment file is detected when deploying a structure using the `Deploy::multi_deploy` function, it will deploy to all provided chains without asking for approval.
+
+## Logging
+
+### RUST_LOG
+
+Optional, accepted values : `debug`, `error`, `info`, `warn`,  `trace`
+
+ `RUST_LOG` defines the Log level for the application. This is actually a `Rust` flag that we use inside `cw-orch`. If working with this environment variable, don't forget to also initialize a logger at the beginning of you application to be able to see the output. You can work with <a href="https://crates.io/crates/pretty_env_logger/" target="_blank">pretty_env_logger</a> for instance.
+
+### CW_ORCH_SERIALIZE_JSON
+
+Optional, accepted values : `false`, `true`
+
+If equals to `true`, in the output logs, cw-orch will serialize the contract messages (instantiate, execute, query,... ) as JSON. This replaces the standard Rust Debug formatting and allows for easy copying and sharing of the executed messages.
+
+### CW_ORCH_DISABLE_ENABLE_LOGS_MESSAGE
```

Comment:
```suggestion
### CW_ORCH_DISABLE_LOG_MESSAGES
```
Just realized that this name is a little confusing.

Reply:
This ? `CW_ORCH_DISABLE_LOGS_ACTIVATION_MESSAGE`

Reply:
Yeah, nice!